cli/core/findStalePods.go

```diff
@@ -85,10 +100,16 @@ func isEigenpod(eth *ethclient.Client, chainId uint64, eigenpodAddress string) (
 		return false, fmt.Errorf("PodOwnerShares() failed: %s", err.Error())
 	}
 
+	balance, err := eth.BalanceAt(context.Background(), common.HexToAddress(eigenpodAddress), nil)
+	if err != nil {
+		return false, fmt.Errorf("balance check failed: %s", err.Error())
+	}
+
 	// Simulate fetching from contracts
 	// Implement contract fetching logic here
 	cache.PodOwnerShares[eigenpodAddress] = PodOwnerShare{
 		Shares:     podOwnerShares.Uint64(),
+		NativeETH:  phase0.Gwei(balance.Uint64()),
```

Comment:
need to divide by 1e9, balanceAt gives wei

Reply:
phenomenal catch